manifests/mod/geoip.pp

```diff
@@ -0,0 +1,29 @@
+class apache::mod::geoip (
+  $enable                  = 'Off',
```

Comment:
i'd rather this was a boolean, then we can just use `bool2httpd()` in the template